src/org/omegat/gui/dictionaries/IDictionariesPane.java

```diff
@@ -25,7 +25,7 @@
 
 package org.omegat.gui.dictionaries;
 
-public interface IDictionaries {
+public interface IDictionariesPane {
```

Comment:
There are `IDictionary` (from core.dictionaries) and `IDictionaries` (from gui.dictionaries)
It is easy for me to confuse it. So I'd like to make this more obvious name.